cmd/agent/subcommands/status/command.go

```diff
@@ -150,15 +146,12 @@ func requestStatus(config config.Component, cliParams *cliParams) error {
 		v.Set("format", "text")
 	}
 
-	endpoint, err := apiutil.NewIPCEndpoint(config, "/agent/status")
-	if err != nil {
-		return err
-	}
+	return v
+}
 
-	res, err := endpoint.DoGet(apiutil.WithValues(v))
-	if err != nil {
-		return err
-	}
+// return "[REDACTED] - failure to clean the message"
+func postProcessRec(res []byte, cliParams *cliParams) error {
```

Comment:
I'm not fully clear `postProcessRec` is the good name for this function. What about `renderResponse` or `printResponse` ?

Reply:
Agree ! `renderResponse` is a better name.
Done in 8b18784111ee1f1fe50fb0d2250c7637a81a1db5